drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -289,6 +289,10 @@ static int lis2dh_acc_config(const struct device *dev,
 	case SENSOR_ATTR_SLOPE_TH:
 	case SENSOR_ATTR_SLOPE_DUR:
 		return lis2dh_acc_slope_config(dev, attr, val);
+#endif
+#ifdef CONFIG_LIS2DH_ACCEL_HP_FILTERS
+	case SENSOR_ATTR_CONFIGURATION:
+		return lis2dh_acc_hp_filter_set(dev, val->val1);
```

Comment:
Is it correct to use this attribute ("Configure the operating modes of a sensor")?                                                                                 
I wonder whether it would be better to use a new one for FILTERs instead of this that seems to be too generic. What's your opinion?

Reply:
@avisconti I also wasn't very keen on using this one, as it it's definitely too generic, but was the closest one that fits into this case. Since "setting filters" is a very unique and rare feature that's not widely available into most of the sensors, I was very sparing in introducing a new one and increasing the attribute list. If in the future an ambiguity arisen, we could specify it more precisely with the cost of adding a new one. Anyway if you still in favor to do it now, and if there isn't anyone against it we could add it. Also any suggestion for the name of such attribute also would be in favor, thanks !

Reply:
> @avisconti I also wasn't very keen on using this one, as it it's definitely too generic, but was the closest one that fits into this case. Since "setting filters" is a very unique and rare feature that's not widely available into most of the sensors, I was very sparing in introducing a new one and increasing the attribute list. If in the future an ambiguity arisen, we could specify it more precisely with the cost of adding a new one. Anyway if you still in favor to do it now, and if there isn't anyone against it we could add it. Also any suggestion for the name of such attribute also would be in favor, thanks !

ok, if suc a case will arise we will add a new one.